app.py

```diff
@@ -77,6 +77,28 @@ async def index(
     return response
 
 
+@app.get("/feed_info", response_class=PrettyJSONResponse, include_in_schema=False)
+async def feed_info(
+    request: Request, x_original_forwarded_for: str | None = Header(default=None)
+):
+    """
+    Return feed info with the client numbers.
+    """
+    client_ip = x_original_forwarded_for
+    clients_beast = provider.get_clients_per_client_ip(client_ip)
+    clients_mlat = provider.mlat_clients_to_list(client_ip)
+    context = {
+        "clients_beast": clients_beast,
+        "clients_mlat": clients_mlat,
+        "own_mlat_clients": len(clients_mlat),
```

Comment:
is 
"own_beast_clients": len(clients_beast),
also useful?

Reply:
I have no clue.
I don't have a feeder IP therefore I cant see even what the data looks like.
I don't know what the difference between mlat client and beast client is.